charts/newrelic-infra-operator/templates/configmap.yaml

```diff
@@ -1,8 +1,24 @@
 {{- define "fargate-config" -}}
 infraAgentInjection:
   resourcePrefix: {{ template "newrelic-infra-operator.fullname" . }}
+{{- if include "newrelic.fargate" . }}
+  agentConfig:
+{{- if not .Values.config.infraAgentInjection.agentConfig.customAttributes }}
+    customAttributes:
+      - name: computeType
+        defaultValue: serverless
+      - name: fargateProfile
+        fromLabel: eks.amazonaws.com/fargate-profile
+{{- end -}}
+{{- if not .Values.config.infraAgentInjection.agentConfig.policies }}
```

Comment:
I think those conditions are not needed, merge should retain the values from config?

Reply:
I think it is safer and more explicit to show that if defined we are not trying to inject them

Reply:
Fair :+1: